lib/dor/models/publishable.rb

```diff
@@ -69,9 +69,10 @@ def publish_metadata
 
     # Call dor services app to have it publish the metadata
     def publish_metadata_remotely
-      dor_services = RestClient::Resource.new(Config.dor_services.url + "/v1/objects/#{pid}/publish")
-      dor_services.post ''
-      dor_services.url
+      dor_services = Dor::Config.dor_services.rest_client
+      endpoint = dor_services["v1/objects/#{pid}/publish"]
+      endpoint.post ''
+      endpoint.url
```

Comment:
It might help to add commentary on why the path construct is appended like this, because this construct may depend on the HTTP client library.  In the code removed, it's clear that `RestClient` is the library in use.  The new code only implies this by using `Config.dor_services.rest_client`.